ingestor/core/config.py

```
@@ -0,0 +1,5 @@
+import yaml
+
+def load_config(path):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)